code/modules/mob/living/carbon/human/species/station/station.dm

```diff
@@ -412,7 +412,7 @@
 
 	light_range = 2
 	light_power = 0.5
-
+	meat_type = /obj/item/stack/material/steel
```

Comment:
I dont think thats going to work with the gibber, because there the meat is used that way:

`var/obj/item/weapon/reagent_containers/food/snacks/meat/new_meat = new slab_type(src, rand(3,8))
if(istype(new_meat))`

Reply:
Also one of the reasons why using the synth check was a better idea, I guess.

Reply:
It should be fixed for real now.